packages/app/src/systems/Account/components/BalanceWidget/BalanceWidget.tsx

```diff
@@ -0,0 +1,183 @@
+import { Avatar, Box, Flex, Icon, IconButton, Text } from "@fuel-ui/react";
+import type { ReactNode } from "react";
+
+import type { Account } from "../../types";
+
+import { BalanceWidgetLoader } from "./BalanceWidgetLoader";
+
+import { shortAddress } from "~/systems/Core";
+
+type BalanceWidgetWrapperProps = {
+  children: ReactNode;
+};
+
+export function BalanceWidgetWrapper({ children }: BalanceWidgetWrapperProps) {
+  const backgroundProps = {
```

Comment:
We should follow the styling pattern used here: r/packages/app/src/systems/Asset/components/AssetItem/AssetItem.tsx